examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -278,7 +282,13 @@
       "name": "JENKINS_IMAGE_STREAM_TAG",
       "displayName": "Jenkins ImageStreamTag",
       "description": "Name of the ImageStreamTag to be used for the Jenkins image.",
-      "value": "jenkins:latest"
+      "value": "jenkins:2"
```

Comment:
I think `jenkins:latest` is ok, no need change this. Because the 'latest' tag in jenkins imagestream is also pointing to jenkins:2

Reply:
It does today. If we change it in the future I don't want to break people. Pointing to latest is generally bad practice unless you are very tolerant of major version changes.

Reply:
OK, understood, better to add a variable like `jenkins:<version>` , what do you think